### DISCUSSION TITLE
Docker Image version still not up-to-date

### DISCUSSION THREAD

**Discussion by laweschan** [alibaba/higress#1500] [General]
Hi,

    I want to start test higress in docker / podman environment, however, follow the instruction here the source image path higress-registry.cn-hangzhou.cr.aliyuncs.com/higress/all-in-one:latest is not refer to the latest (currently should be 2.0.3) , may I know is that the correct image to follow ?

    thanks

Regards,
Lawes

**Comment by CH3CHO** [comment]
Please try again.

![image](https://github.com/user-attachments/assets/9e81184f-3210-435e-8554-0dbcb9b9ec17)
